### ISSUE TITLE
vmalert-tool: process won't exit when no rule is defined under group

### ISSUE THREAD

**Issue by Haleygo** [VictoriaMetrics/VictoriaMetrics#6500]
### Is your feature request related to a problem? Please describe

Run `./vmalert-tool unittest -files  .test1.yaml"
with test1.yaml
```
rule_files:
  - rules.yaml

evaluation_interval: 1m

tests:
  - interval: 1m
    name: "basic test"
    input_series:
      - series: "test"
        values: "_x5 1x5 _ stale"

    alert_rule_test:
      - eval_time: 1m
        groupname: group1
        alertname: SameAlertNameWithDifferentGroup
        exp_alerts: []
```
and rules.yaml
```
groups:
  - name: group1
    rules: []
```
and process won't exit.

### Describe the solution you'd like

vmalert-tool exits normally.

**Comment by hagen1778** [comment]
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6502 has been merged and will be available in the next release.

**Comment by valyala** [comment]
This bug must be fixed in [v1.102.0-rc2 release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc2).

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been included in [v1.97.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.6).
